src/main/java/edu/rosehulman/covidtracer/model/Person.java

```diff
@@ -51,9 +59,16 @@ public Person(String firstName, String middleName, String lastName, String roseI
 		this.housingLocation = housing;
 	}
 	
-	public Person(Long id, String firstName, String middleName, String lastName, String roseID, String email, String phone, String housing) {
-		this(firstName, middleName, lastName, roseID, email, phone, housing);
+	public Person(Long id, String firstName, String middleName, String lastName, String roseID, String email, String phone, String housing, Role role) {
```

Comment:
Why did we get rid of the this(...) call?

Reply:
Not sure, I think I was confused or something. I'll change that back